app/jobs/school_data_importer_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SchoolDataImporterJob < CronJob
+  self.cron_expression = "0 6 * * *"
```

Comment:
Just wondering if this should be moved to a constant or even to a config item.

Reply:
Eh, maybe? I reckon doing it when we have a single job to keep track of is a bit of an overkill. Once we have many, and we want them to happen in particular order, I think it would make sense to have a single file specifying that.

Reply:
I was thinking more along the lines of not having to do a pr when some stakeholder says run the import at 8 am on Saturdays instead. In the config we would just update it but in the code it would be more involved.

Reply:
Well, thing is, this config value would need to have a default, and if the stakeholder was important enough, then we probably would have to change the default value as well. 

I'm happy to move it to config once we get such a request though :)

Reply:
We can leave it for now. Anyway, the default would be what we think is reasonable and the value would be what the stakeholder requests.

Also, for now running this at 6 in the morning might be ok but on a bust site we'd probably wan't to do the run between 12-4am to avoid slow performance for users.